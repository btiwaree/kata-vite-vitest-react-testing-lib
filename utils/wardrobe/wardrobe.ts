// 50cm, 75cm, 100cm, and 120cm
// max 250cm length

const sizes = [50, 75, 100, 120];
const maxSize = 250;

50 50 50 50 50 = 250 store [50 50 50 50 50] break;
75 75 75 75 = 300 break;
100 100 100 = 300 break;
120 120 120 = 360 break;


export const allCombinations = () => {
  return [];
};
