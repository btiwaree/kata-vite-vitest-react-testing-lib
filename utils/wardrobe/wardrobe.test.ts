import { describe, it } from 'vitest';

import { allCombinations } from './wardrobe.ts';

describe('wardrobe', () => {
  it('allCombination', () => {
    const combinations = allCombinations();

    expect(combinations).toEqual([]);
  });
});
