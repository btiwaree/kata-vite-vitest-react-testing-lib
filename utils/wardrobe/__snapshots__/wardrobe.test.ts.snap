// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`wardrobe > allCombination 1`] = `undefined`;
